api-services/offline/users/organization_service.tsx

```diff
@@ -153,22 +158,6 @@ export class OrganizationOfflineServicePromiseClient extends OrganizationService
     return new CreateOrganizationResponse().setId(newOrganization.id)
   }
 
-  async createOrganizationForUser(request: CreateOrganizationForUserRequest, _?: Metadata): Promise<CreateOrganizationForUserResponse> {
-    const newOrganization: OrganizationMinimalDTO = {
-      id: Math.random().toString(),
-      shortName: request.getShortName(),
-      longName: request.getLongName(),
-      avatarURL: 'https://helpwave.de/favicon.ico',
-      email: request.getContactEmail(),
-      isPersonal: request.getIsPersonal(),
-      isVerified: true
-    }
-
-    OrganizationOfflineService.create(newOrganization)
-
-    return new CreateOrganizationForUserResponse().setId(newOrganization.id)
-  }
-
```

Comment:
Removing this will have the effect that super is called and the GRPC client makes a backend request. For correct offline behavior, this method should stay implemented

Reply:
Removed since https://github.com/helpwave/services/commit/12e8933752cf54502dbecfb72833f01060ccccdd#diff-5c30cc40fa91c587bf380a97a8624fd62d22fed92d8580f6b315c2cf2a45500cL34